packages/transport-server/src/node-cursor.spec.ts

```diff
@@ -140,25 +140,6 @@ describe('NodeCursor', () => {
     });
   });
 
-  describe('#explain', () => {
```

Comment:
Explain has to be implemented and can't return a cursor since the driver returns results